src/com/janek/lib/math/MathUtilTest.java

```diff
@@ -63,6 +78,62 @@ void divideNumbersTest() {
         assertEquals(2, result);
     }
 
+    @Test
+    @DisplayName("power - if its calculating correctly")
```

Comment:
Please do the test description more more clearly, e.g

power - when exponent is 0 - should return 0